decidim-debates/app/views/decidim/debates/debates/show.html.erb

```diff
@@ -54,11 +54,14 @@ edit_link(
   <% end %>
 
   <section class="layout-main__section layout-main__buttons" data-buttons>
-    <% if endorsements_enabled? && allowed_to?(:endorse, :debate, debate: debate) %>
-      <%= endorsement_buttons_cell(debate) %>
+  <% if endorsements_enabled? %>
+    <% if allowed_to?(:endorse, :debate, debate: debate) %>
+        <%= endorsement_buttons_cell(debate) %>
+      <% else %>
+        <%= endorsers_list_cell(debate) %>
     <% end %>
+  <% end %>
     <%= cell "decidim/comments_button", nil %>
```

Comment:
Let's leave the break line here to separate the three elements of this part of the page: endorsements, comments, and tags 
```suggestion
    <%= cell "decidim/comments_button", nil %>

```